core/src/main/scala-2.13/org/allenai/common/ScalaUtils.scala

```diff
@@ -0,0 +1,3 @@
+package org.allenai.common
+
+object ScalaUtils {}
```

Comment:
This is specifically for 2.13. I'm not sure the method in here was even used, but I don't think we'll need it for 2.13. Specifically because `mapValues` is deprecated.